app/views/users/profile.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Profile:</h1>
+
+<h3>Username:<%=' '@user.username%></h3>
+<h3>Your email:<%=' '@user.email%></h3>
+
+<p>This is where we could display stats such as number of questions asked, votes, and comments as well as a number of best answers. Maybe. That'd be later. Also wondering if there's really a point to this profile? Maybe links to your questions? Yeah, that would make sense. Part of the project or something to icebox for when we're done?</p>
+
```

Comment:
👍 
I agree that we don't need a user profile right now but definitely something to include as a stretch goal. Good thinking.